qutebrowser/mainwindow/tabwidget.py

```diff
@@ -275,6 +306,7 @@ def __init__(self, win_id, parent=None):
         self._auto_hide_timer.timeout.connect(self._tabhide)
         self.setAutoFillBackground(True)
         self.set_colors()
+        self.pinned = 0
```

Comment:
I'd prefer this to be called `pinned_count` or so, otherwise it sounds like it'd be a boolean.